src/main/java/Duke.java

```diff
@@ -1,10 +1,127 @@
+            catch (Exception e) {
+                System.out.println("Cannot parse deadline start date! Requires [/by]");
+            }
+            list[taskLength] = new Deadline(taskFinal);
+            break;
+        case "event":
+            try {
+                String eventTiming = taskContent.split("/from ")[1];
+                String eventFrom = eventTiming.split("/to ")[0];
+                String eventTo = eventTiming.split("/to ")[1];
+                taskFinal = taskContent.split("/")[0] + " (from: " + eventFrom + "to: " + eventTo + ")";
+            }
+            catch (Exception e) {
+                System.out.println("Cannot parse event start/end date! Requires [/from /to]");
+            }
+            list[taskLength] = new Event(taskContent);
+            break;
+        default:
+            System.out.println(taskType);
+            list[taskLength] = new Todo(taskContent);
+        }
+        taskLength++;
+        System.out.println("____________________________________________________________\n"
+                + "Okay, I've added: " + taskFinal + "\n"
+                + "____________________________________________________________");
+    }
+
+    //show current tasks
+    public static void showTasks() {
+        System.out.println("____________________________________________________________");
+        System.out.println("Your current tasks");
+        for (int i = 0; i < taskLength; i++){
+            System.out.print((i + 1) + ". ");
+            list[i].showTask();
+            //System.out.println(list[i].classSpecifics());
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void mark(int taskNumber){
+        System.out.println("Good job! I've marked task " + taskNumber + " as done");
+        list[taskNumber - 1].markAsDone();
+    }
+
+    public static void unMark(int taskNumber){
+        System.out.println("I've marked task " + taskNumber + " as not done");
+        list[taskNumber - 1].markNotDone();
+    }
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String chat_name = "Sigma";
+        String output = "____________________________________________________________\n"
+                + " Hello! I'm " + chat_name + "\n"
+                + " What can I do for you?\n"
+                +"____________________________________________________________\n";
+        System.out.println(output);
+        while (true) {
+            Scanner in = new Scanner(System.in);
+            String input = in.nextLine();
+            if (input.equals("bye")) {
+                break;
+            }
+            else if (input.equals("list")) {
```

Comment:
small fix, when doing if else statements, you dont want to put else if on another line. You would want to put else if after the } 

looks like:
if {
} else if {
} else {
}